sdk/cognitivelanguage/Azure.AI.Language.Conversations/README.md

```diff
@@ -1,56 +1,56 @@
 # Azure Cognitive Language Services Conversations client library for .NET
 
-The Question Answering service is a cloud-based API service that lets you create a conversational question-and-answer layer over your existing data. Use it to build a knowledge base by extracting questions and answers from your semi-structured content, including FAQ, manuals, and documents. Answer users’ questions with the best answers from the QnAs in your knowledge base—automatically. Your knowledge base gets smarter, too, as it continually learns from user behavior.
+Conversations (aka LuisVNext) is a cloud-based conversational AI service that applies custom machine-learning intelligence to a user's conversational, natural language text to predict overall meaning, and pull out relevant, detailed information. The service utilizes state-of-the-art technology to create and utilize natively multilingual models, which means that users would be able to train their models in one language but predict in others. 
 
-[Source code][questionanswering_client_src] | [Package (NuGet)][questionanswering_nuget_package] | [API reference documentation][questionanswering_refdocs] | [Product documentation][questionanswering_docs] | [Samples][questionanswering_samples]
+[Source code][conversationanalysis_client_src] | [Package (NuGet)][conversationanalysis_nuget_package] | [API reference documentation][conversationanalysis_refdocs] | [Product documentation][conversationanalysis_docs] | [Samples][conversationanalysis_samples]
 
 ## Getting started
 
 ### Install the package
 
-Install the Azure Cognitive Language Services Question Answering client library for .NET with [NuGet][nuget]:
+Install the Azure Cognitive Language Services Conversations client library for .NET with [NuGet][nuget]:
 
 ```powershell
-dotnet add package Azure.AI.Language.QuestionAnswering --prerelease
+dotnet add package Azure.AI.Language.Conversations --prerelease
 ```
 
 ### Prerequisites
 
 * An [Azure subscription][azure_subscription]
-* An existing Question Answering resource
+* An existing Text Analytics resource
```

Comment:
Is "Text Analytics" here right? Is it the same resource type? At the very least, we may want to point that out, but I believe they are different resource types.

Reply:
When I use the CLU Language Studio (the one in preview) to create new projects, I need to choose an Azure language resource of type "Text Analytics". I am not sure if this will still be the case after preview is over, I'll check and let you know.

Reply:
I checked with a service team privately, and it seems like this will remain to be the case.